rubygem-scaptimony/rubygem-scaptimony.spec

```diff
@@ -0,0 +1,118 @@
+# This package contains macros that provide functionality relating to
+# Software Collections. These macros are not used in default
+# Fedora builds, and should not be blindly copied or enabled.
+# Specifically, the "scl" macro must not be defined in official Fedora
+# builds. For more information, see:
+# http://docs.fedoraproject.org/en-US/Fedora_Contributor_Documentation/1/html/Software_Collections_Guide/index.html
+
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name scaptimony
+
+%global mainver 0.2.0
+%global release 1
+%{?prever:
+%global gem_instdir %{gem_dir}/gems/%{gem_name}-%{mainver}%{?prever}
+%global gem_docdir %{gem_dir}/doc/%{gem_name}-%{mainver}%{?prever}
+%global gem_cache %{gem_dir}/cache/%{gem_name}-%{mainver}%{?prever}.gem
+%global gem_spec %{gem_dir}/specifications/%{gem_name}-%{mainver}%{?prever}.gemspec
+}
+
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: %{mainver}
+Release: %{release}%{?dist}
+Summary: SCAPtimony is SCAP database and storage server
+Group: Applications/System
+License: GPLv3
+URL: https://github.com/OpenSCAP/scaptimony
+Source0: http://rubygems.org/downloads/%{gem_name}-%{version}.gem
+
+%if 0%{?fedora} > 18
+Requires: %{?scl_prefix}ruby(release)
```

Comment:
Just another thing that depends on a specific Rails version, but now I see it's a full blown engine, so I'm not surprised.